packages/react-components/semantic-style-hooks-preview/library/README.md

```diff
@@ -3,3 +3,32 @@
 **Semantic Style Hooks components for [Fluent UI React](https://react.fluentui.dev/)**
 
 These are not production-ready components and **should never be used in product**. This space is useful for testing new components whose APIs might change before final release.
+
+** Semantic Token Migration Guide **
+
+The Semantic Style Hooks package will enable opt-in behavior for semantic token style overrides.
```

Comment:
The phrasing on these two sentences tripped me up.

You might consider branding this whole package "Semantic Token Theme." - "Semantic Theme" is a little less clear to me. Can themes be semantic?

We can drive home the party line that "A theme is a combination of token value changes, some additional net new tokens and custom style hooks"

Either way, very grateful for this document and the deprecation plan.

Reply:
I've modified the text here to be a bit more specific; we'll likely end up with an actual Semantic Token Theme package on top of this so trying to avoid merging the concepts for now.

Currently thinking of the packages as:

**Semantic Tokens**: New CSSVar and variable definitions set w/ legacy v9 fallbacks

**Semantic Style Hooks**: Enables semantic tokens on core Fluent components, which enables both Legacy and Semantic themes via tokens.

**Semantic Theme**: The key/value sets for semantic token definitions (similar to fluentui/react-theme), includes Theme tooling (creation/optimization functions).

These packages should work together and enable each other so I'll think some more on how to market it as a singular concept.